src/Compilers/Core/Portable/InternalUtilities/ReflectionUtilities.cs

```diff
@@ -15,6 +15,9 @@ internal static class ReflectionUtilities
     {
         private static readonly Type Missing = typeof(void);
 
+#if NET6_0_OR_GREATER
+        [RequiresUnreferencedCode("Loading arbitrary type by name")]
```

Comment:
Why didn't you follow the `TrimWarnings.` pattern here? 

Assume I said this for all the other cases where you hard code a string in the change :smile:

Reply:
Same as above.